crates/api_crud/src/post/update.rs

```diff
@@ -70,15 +72,15 @@ pub async fn update_post(
     Err(LemmyErrorType::NoPostEditAllowed)?
   }
 
-  // Fetch post links and Pictrs cached image if url was updated
+  // Fetch post links and thumbnail if url was updated
   let (embed_title, embed_description, embed_video_url, thumbnail_url) = match &url {
     Some(url) => {
       let metadata = fetch_link_metadata(url, true, &context).await?;
       (
         Some(metadata.opengraph_data.title),
         Some(metadata.opengraph_data.description),
         Some(metadata.opengraph_data.embed_video_url),
-        Some(metadata.thumbnail),
+        Some(custom_thumbnail.map(Into::into).or(metadata.thumbnail)),
```

Comment:
If you edit the text of a post, this will likely refetch the link thumbnail and overwrite the custom thumbnail.

Reply:
K I think I got this fixed now.